### ISSUE TITLE
Bastok Mission 2-3: Windurst -> San d'Oria

### ISSUE THREAD

**Issue by Annihilus343** [DarkstarProject/darkstar#285]
First bug post, I did most of the mission a few weeks ago up til getting the Scene with Lion in the Consulate at San d'Oria. I am trying to finish now but Helaku nor Halver will give the scenes I need with them.. they won't respond to my interaction at all. I have fought the dragon twice and have no Kindred Crest as I expected as much. Not sure what to do

**Comment by helixhamin** [comment]
With the new changes to Halver, has this been fixed?

**Comment by TeoTwawki** [comment]
This NPC is still broken.
```lua
elseif (missionStatus == 7 or missionStatus == 8) then
```
Invalid variable value.

**Comment by TeoTwawki** [comment]
This NPC is still broken.
```lua
elseif (missionStatus == 7 or missionStatus == 7.5) then
```
Invalid variable value. I think it should be 8 not 7 and certainly not 7.5 (which winds up still being 7), but I don't have time to verify.

**Comment by TeoTwawki** [comment]
The variable was fixed by #2971 so Helaku checks for 7 and then sets 8 after the cs now.
